README.md

```diff
@@ -26,158 +40,78 @@ To install asdf using brew, follow instructions at https://asdf-vm.com . In shor
+echo 'export ANDROID_NDK_HOME="$ANDROID_HOME/ndk/23.1.7779620"' >> ~/.bashrc
+```
 
-Start a new terminal to get the changes to the environment.
+Start a new terminal to get the changes to the environment .
 
-(optional) To install Android Studio, download the latest
+To install Android Studio, download the latest
 android-studio-{version}-linux.tar.gz from
 https://developer.android.com/studio . (Tested with Jellyfish 2023.3.1 .)
 In a terminal, enter the following with the correct {version}:
 
-    sudo tar -C /usr/local -xzf android-studio-{version}-linux.tar.gz
-
-To launch Android Studio, in a terminal enter:
-
-    /usr/local/android-studio/bin/studio.sh &
-
-### Get a copy of the repo
-
-```console
-git clone https://github.com/gnolang/gnonative
-cd gnonative
-```
-
-### Build for Android
-
-#### Set up the Android NDK
-
--   Launch Android Studio and accept the default startup options. Create a new
-    default project (so that we get the main screen).
--   On the Tools menu, open the SDK Manager.
--   In the "SDK Tools" tab, click "Show Package Details". Expand
-    "NDK (Side by side)" and check "23.1.7779620".
--   Click OK to install and close the SDK Manager.
-
-#### Install the tools with asdf (only need to do once)
-
-(If not building for iOS, edit the file `.tool-versions` and remove the unneeded lines for `ruby` and `cocoapods`.)
-
-```console
-make asdf.install_tools
-```
-
-#### Build the Go code as a library
-
-```console
-make build.android
+```sh
+sudo tar -C /usr/local -xzf android-studio-{version}-linux.tar.gz
 ```
 
-#### Start metro
-
-```console
-cd examples/js/react-native/gnoboard
-npm start
-```
-
-#### Connect your Android phone/emulator and bind its port to metro
-
-You can either connect an Android phone via USB cable, or launch an emulator device from Android Studio.
-
-##### Real device
-
-Connect your device and bind the port to metro:
-
-```console
-cd examples/js/react-native/gnoboard
-make android.reverse_tcp
-```
-
-##### Emulator device
-
-You can either run Android Studio and open the Android project in `examples/js/react-native/gnoboard/android`.
-If you prefer the CLI option:
-
-```console
-android-studio ./android
-```
-
-Once done, bind the port to metro:
+To launch Android Studio, in a terminal enter:
 
-```console
-make android.reverse_tcp
+```sh
+/usr/local/android-studio/bin/studio.sh &
 ```
 
-#### Build with Android Studio
-
-Open Android Studio and open the current Android project if it's not already done.
-Select the right device in the device list. Open the `Run` menu, and select `Run app`.
-See more: https://developer.android.com/studio/run#basic-build-run
-
-### Build for iOS
-
-#### Install the tools with asdf (only need to do once)
+### Install the tools with asdf (only need to do once)
 
-```console
+```sh
 make asdf.install_tools
 ```
 
 If you get an error like "https://github.com/CocoaPods/CLAide.git (at master@97b765e) is not yet checked out" then reinstall cocoapods like this:
```

Comment:
This error can only happen when building iOS. Previously, it was under the header "Build fo iOS". It's OK if you want to remove the header. Maybe change "If you get an error like" to "On macOS, if you get an error like".